server/app.ts

```diff
@@ -71,7 +72,14 @@ const schema = makeExecutableSchema({ typeDefs,
 	resolvers
 });
 
-app.use('/graphql', bodyParser.json(), graphqlExpress({ schema }));
+app.use(
+	'/graphql',
+	bodyParser.json(),
+	isAdmin,
+	graphqlExpress({
+		schema
+	})
+);
 app.use('/graphiql', graphiqlExpress({ endpointURL: '/graphql' }));
```

Comment:
Will we remove graphiql during production? Maybe we can include a dev/production mode in the future.

Reply:
No, we usually keep graphiql around even in production. It will still be blocked if it doesn't have the right key because all that page does is call `/graphql` endpoints.